pandas/core/arrays/datetimelike.py

```diff
@@ -7,11 +7,11 @@
 import numpy as np
 
 from pandas._libs import NaT, NaTType, Timestamp, algos, iNaT, lib
+from pandas._libs.tslibs._timestamp import maybe_integer_op_deprecated
```

Comment:
possibly put `maybe_integer_op_deprecated` into `tslibs.__init__` to avoid the private import?